cmds/date/date.go

```diff
@@ -172,23 +172,23 @@ func getTime(s string) (t time.Time, err error) {
 	year := time.Now().Year() % 100
 	century := time.Now().Year() / 100
```

Comment:
Wouldn't it be `century` be more complicated than that? `2001 / 100 = 20,01` and not 21 for example. Am I missing something?